src/Field/AbstractInputField.php

```diff
@@ -18,7 +19,7 @@
 abstract class AbstractInputField implements FieldInterface
 {
 
-    use FieldsArgumentsAwareObjectTrait, AutoNameTrait;
+    use FieldsArgumentsAwareObjectTrait, AutoNameTrait, ResolvableObjectTrait;
```

Comment:
I took everything except of this line... why InputField need ResolvableObjectTrait?